app/styles/atoms/badge.less

```diff
@@ -155,3 +171,28 @@
 .upf-badge.upf-badge--extended-violet.upf-badge--plain {
   .generate-extended-palette-plain-badge(violet);
 }
+.upf-badge.upf-badge--extended-smart {
+  background: var(--color-smart-gradient-50) padding-box, var(--color-smart-gradient-500) border-box;
+  border-radius: 50em;
```

Comment:
50em ?? 🤔 (why that unit ??)

Reply:
I tried many configurations to make this works and stopped to the first one working, but it seems 50% works too indeed